### DISCUSSION TITLE
WebView.Avalonia repo is not available anymore due to violation of GitHub’s Trademark policy

### DISCUSSION THREAD

**Discussion by antoniovalentini** [Tyrrrz/YoutubeDownloader#552] [Q&A]
I see you're using the `WebView.Avalonia`, but recently [the repo](https://github.com/MicroSugarDeveloperOrg/Avalonia.WebView) has been disabled. Do you know what might have happened?

Also, by looking at the `.csproj`, I noticed you also reference `Avalonia.WebView` as TrimmerRootAssembly. I can't find the latter on the Nuget repo. Is there a repo I can have a look at?

**Comment by Tyrrrz** [comment]
I had no knowledge of that, thanks for the tip. I'll look into it.

> Also, by looking at the .csproj, I noticed you also reference Avalonia.WebView as TrimmerRootAssembly. I can't find the latter on the Nuget repo. Is there a repo I can have a look at?

The package is called `WebView.Avalonia`, but the actual assemblies have different other names, including `Avalonia.WebView`. They weren't working well with trimming so I had to selectively disable it for those assemblies.
